run_all_tests.sh

```diff
@@ -1,3 +1,3 @@
 #!/usr/bin/env bash
 export SBT_OPTS="-XX:+CMSClassUnloadingEnabled -XX:MaxMetaspaceSize=1G"
-sbt clean compile coverage test it:test component:test coverageOff coverageReport
+sbt -Dlogger.application=INFO clean compile coverage test it:test component:test coverageOff coverageReport
```

Comment:
Did you mean to add this logger config?

Reply:
It's added to the test logging xml. This way when running tests for coverage we get coverage of logger.info but when we simply run sbt test we don't get all the verbose info logs.